### PR TITLE
Fix for SAIC-186.

### PR DESCRIPTION
**Pull request by raiesmh08**
Currently log configuration file is hardcoded in
'CloudFerry/cloudferrylib/utils/utils.py'.
Instead log config should be available for configuration from the
general CloudFerry configuration file.

Implementation of will go in 2 part -

1. Logging customization
2. Hard coding removal

For the time this part implements point 1.

### CODE REVIEW

**Comment by raiesmh08** [comment]
In progress

File: cloudferrylib/utils/utils.py [hunk 1]
```diff
@@ -32,6 +32,10 @@
 import yaml
 from logging import config
 
+import cfglib
+
+# Initialize default config
+cfglib.initialize()
```
Comment (roman-verchikov) [inline cloudferrylib/utils/utils.py]: What if `utils.py` never gets imported? I would suggest that you move all the logging-related logic into a separate module/class.

**Comment by roman-verchikov** [comment]
Please provide description for the new option in `configs/config.ini`

**Comment by roman-verchikov** [comment]
Please update commit message to follow pattern:
```
<One-line title describing the feature/bugfix>

<multiline description of what was done>
```

**Comment by raiesmh08** [comment]
Leaving it for the time. If time permits I will be looking it soon :)

**Pull request closed without merging**